### PR TITLE
EDGECLOUD-3609 EDGECLOUD-3625 EDGECLOUD-3246 GPU hostname override, TLS support, orgName update

### PR DESCRIPTION
**Pull request by brucearmstrong**
- Builder support for Cloudlet class, standardized some member variable names.
- Custom hostname can be used for GPU CV activities if the override preference is enabled. 
- TLS support for speedtest and CV image sending, both https and wss.
- Bug fixes for GPU HA support.

### CODE REVIEW

File: android/MobiledgeXSDKDemo/computervision/src/main/java/com/mobiledgex/computervision/ImageProcessorFragment.java [hunk 1]
```diff
@@ -1338,10 +1365,19 @@ private boolean getAppInstList() throws InterruptedException, ExecutionException
                 //Find fqdnPrefix from Port structure.
                 String fqdnPrefix = "";
                 List<Appcommon.AppPort> ports = appInst.getPortsList();
-                for (Appcommon.AppPort aPort : ports) {
+                // Get data only from first port.
+                Appcommon.AppPort aPort = ports.get(0);
```
Comment (lgarner) [inline android/MobiledgeXSDKDemo/computervision/src/main/java/com/mobiledgex/computervision/ImageProcessorFragment.java:1369]: There's only one port (and the first one?)

File: android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MainActivity.java [hunk 2]
```diff
@@ -1074,7 +1085,17 @@ public void run() {
                     } else {
                         Log.i(TAG, "addMarker for "+cloudletName);
                         marker = mGoogleMap.addMarker(new MarkerOptions().position(latLng));
-                        cloudlet = new Cloudlet(cloudletName, appName, carrierName, latLng, distance, fqdn, marker, FQDNPrefix, publicPort);
+                        cloudlet = new CloudletBuilder()
```
Comment (lgarner) [inline android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MainActivity.java:1088]: great. enforce type, though most aren't optional.

**Review by lgarner — APPROVED** [review]
LGTM.

File: android/MobiledgeXSDKDemo/computervision/src/main/java/com/mobiledgex/computervision/ImageProcessorFragment.java (hunk 1 above)
Comment (brucearmstrong) [reply android/MobiledgeXSDKDemo/computervision/src/main/java/com/mobiledgex/computervision/ImageProcessorFragment.java:1369]: There should be 2 ports in the ComputerVision app definition: 8008 for REST and WS. 8009 for Persistent_Socket, my made-up protocol which probably should go away at some point. We only care about the first port for determining the prefix and whether we are TLS.

**Review by ah1053 — APPROVED** [review]
⭐

**Pull request merged**